src/modal/modal-ref.ts

```diff
@@ -5,17 +5,36 @@ import { takeUntil } from 'rxjs/operators';
 
 import { NgbModalBackdrop } from './modal-backdrop';
 import { NgbModalWindow } from './modal-window';
+import { NgbModalOptions } from './modal-config';
+import { isDefined } from '../util/util';
 
 import { ContentRef } from '../util/popup';
 import { isPromise } from '../util/util';
 
+export type NgbUpdatableModalOptions = Pick<
```

Comment:
Could we rename it to `NgbModalUpdatableOptions` for consistency with `NgbModalOptions` ?

Reply:
Done